cmd/ns_finder.go

```diff
@@ -74,15 +80,19 @@ func (f NsFinder) GetApp(appName string, stackName string) (*types.Application,
 	return &matched[0], nil
 }
 
-func (f NsFinder) GetAppWorkspace(app *types.Application, envName string) (*types.Workspace, error) {
+func (f NsFinder) GetEnv(stackName, envName string) (*types.Environment, error) {
 	client := api.Client{Config: f.Config}
-
-	env, err := client.EnvironmentsByName().Get(app.StackName, envName)
+	env, err := client.EnvironmentsByName().Get(stackName, envName)
 	if err != nil {
 		return nil, fmt.Errorf("error retrieving environment: %w", err)
 	} else if env == nil {
-		return nil, fmt.Errorf("environment %s/%s does not exist", app.StackName, envName)
+		return nil, fmt.Errorf("environment %s/%s does not exist", stackName, envName)
```

Comment:
If the env can't be found, it should be throwing a 404. This error message should say something about this being unexpected.

Reply:
If a user runs a command `nullstone push arcana pord`, then this error message will tell the user `environment core/pord does not exist`. The only way I can imagine improving is suggesting the environments that do exist in the stack.

Any thoughts?